frontend/src/components/Map/index.js

```diff
@@ -78,13 +79,17 @@ export default function Map({ draggable = true }) {
 
   const addLayers = async (map) => {
     const data = await fetchCovidData(dataScope.ALL);
+    const usData = [data["data"]["adm0"][177]];
+    //world data expcets US for world layer
```

Comment:
I believe this comment is misleading since 
```javascript
const worldData = data["data"]["adm0"];
```
contains all the countries, including US

Reply:
True